gpt_json/types_streaming.py

```diff
@@ -0,0 +1,50 @@
+from enum import Enum
+from typing import Generic, Type, TypeVar
+
+from pydantic import BaseModel
+
+SchemaType = TypeVar("SchemaType", bound=BaseModel)
+
+class StreamEventEnum(Enum):
+    OBJECT_CREATED = "OBJECT_CREATED"
+    KEY_UPDATED = "KEY_UPDATED"
+    KEY_COMPLETED = "KEY_COMPLETED"
+    OTHER = "OTHER"
```

Comment:
This enum value is a bit vague. Seems like it's currently used before we have started streaming values? Can you define what it's intended to be more precisely?